inst/tutorials/01-edibnb/01-edibnb.Rmd

```diff
@@ -195,9 +195,9 @@ quiz(
            answer("Unimodal",
                   correct = TRUE),
            answer("Bimodal",
-                  message = "How many prominent peaks do you see?"),
+                  message = "How many prominent peaks, or modes, do you see? (recall \"bi\" means \"two\")"),
            answer("Multimodal",
-                  message = "How many prominent peaks do you see?"),
+                  message = "How many prominent peaks, or modes, do you see?"),
```

Comment:
Maybe you can add a bit more here like the previous one, e.g. "A distribution is said to be multimodal if there are three or more prominent peaks. How many prominent peaks, or modes, do you see?"